.github/renovate.json

```diff
@@ -70,6 +70,12 @@
     },
     {
       "matchPackageNames": ["moby-runc", "moby-containerd"],
```

Comment:
is moby-runc also used for Ubuntu ?

Reply:
I guess we install is seperately for Ubuntu and it's hidden for AzLinux.

Reply:
For AzLinux,
https://github.com/Azure/AgentBaker/blob/master/parts/linux/cloud-init/artifacts/README-COMPONENTS.md#were-mariner-and-azurelinux-intentionally-removed-from-package-runc